src/test/java/com/jcabi/github/RtContentsITCase.java

```diff
@@ -45,7 +46,8 @@
  * @author Andres Candal (andres.candal@rollasolution.com)
  * @version $Id$
  * @since 0.8
- * @checkstyle MultipleStringLiterals (300 lines)
+ * @checkstyle MultipleStringLiterals (500 lines)
+ * @checkstyle MagicNumberCheck (500 lines)
```

Comment:
Increased MultipleStringLiterals range to account for increased file size.
MagicNumber in question is the number of content entries returned by `iterate()`, which is 1 greater than the number of entries created because of the `readme.md` in root.